Explorer/Assets/DCL/NameTags/NametagView.cs

```diff
@@ -9,6 +15,131 @@ public class NametagView : MonoBehaviour
         public TMP_Text Username { get; private set; }
 
         [field: SerializeField]
-        public GameObject VerifiedIcon { get; private set; }
+        public TMP_Text MessageContent { get; private set; }
+
+        [field: SerializeField]
+        public SpriteRenderer Background { get; private set; }
+
+        [field: SerializeField]
+        public SpriteRenderer BubblePeak { get; private set; }
+
+        [field: SerializeField]
+        public RectTransform MessageContentRectTransform { get; private set; }
+
+        [field: SerializeField]
+        internal AnimationCurve backgroundEaseAnimationCurve { get; private set; }
+
+        [field: SerializeField]
+        internal AnimationCurve alphaOverDistanceCurve { get; private set; }
+
+        public string Id;
+
+        [field: SerializeField]
+        public float FixedWidth { get; private set; }
+
+        private const float NAMETAG_MARGIN_OFFSET_WIDTH = 0.2f;
+        private const float NAMETAG_MARGIN_OFFSET_HEIGHT = 0.15f;
+        private const float BUBBLE_MARGIN_OFFSET_WIDTH = 0.4f;
+        private const float BUBBLE_MARGIN_OFFSET_HEIGHT = 0.6f;
+        private const float ANIMATION_DURATION = 0.7f;
+        private const float FULL_OPACITY_MAX_DISTANCE = 8.5f;
+        private const int CHAT_BUBBLE_IDLE_TIME = 6000;
```

Comment:
That's quite a lot of constants, I would extract them into a ScriptableObject for easier edition in the future